circleci/templates/maven/config_template-no-docker-v1.yml

```diff
@@ -109,13 +116,47 @@ jobs:
       - image: circleci/ruby:2.4.1
     steps:
       - run: echo "*PLACEHOLDER* pretending to execute integration testing phase!"
-  Deploy Artifacts: # Deploy artifacts to Nexus or Maven Central
-    docker:
-      - image: circleci/ruby:2.4.1
-    steps:
-      - run: echo "*PLACEHOLDER* pretending to deploy build artifacts to Maven Central!"
 # END: UNDER CONSTRUCTION AREA ---------------------------------------
 
+  Tag Release:
+    docker: *image
+    steps:
+      # Tag Release creates a Git tag with given a given name and pushes it
+      # This job is governed by the following environment variables:
+      # $RELEASE - Set to true to perform a release, otherwise this step is skipped
+      # $RELEASE_TAG - The release tag for this version
+      # $RELEASE_VERSION - The version to release
+      # $NEXT_VERSION - The next development version
+      # To use this step the following MUST also be set: $GITHUB_KEY and $ION_GPG_KEYRING
```

Comment:
What does ION_GPG_KEYRING do in this case?
I get what the other inputs are because they are straightforward.

Reply:
This is used by the maven gpg plugin, which has a 'sign' goal to sign the generated artifacts during the verify phase when deploying. So the ION_GPG_KEYRING is used in preparing the release and deploying to sign the artifacts.